data-prepper-api/src/test/java/com/amazon/dataprepper/model/event/JacksonEventTest.java

```diff
@@ -299,7 +301,7 @@ public void testKey_withNullKey_throwsNullPointerException() {
     public void testToString_withEmptyData() {
         final String result = event.toJsonString();
 
-        assertThat(result, is(equalTo("{}")));
+        assertThat(result, is(equalTo("{\"event_timestamp\":\"" + JacksonEvent.getZonedDateTime() + "\"}")));
```

Comment:
I'd call `get("event_timestamp")` on the new Event.

Then you can compare against that instead of this static method.